templates/include/glean/client_info.1.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "additionalProperties": false,
+  "properties": {
+    "android_sdk_version": {
+      "type": "string"
+    },
+    "app_build": {
+      "description": "The build identifier generated by the CI system (e.g. \"1234/A\"). For language bindings that provide automatic detection for this value, (e.g. Android/Kotlin), in the unlikely event that the build identifier can not be retrieved from the OS, it is set to \"inaccessible\". For other language bindings, if the value was not provided through configuration, this metric gets set to `Unknown`.",
+      "type": "string"
+    },
+    "app_channel": {
+      "description": "The channel the application is being distributed on.",
+      "type": "string"
+    },
+    "app_display_version": {
+      "description": "The user visible version string (e.g. \"1.0.3\").  In the unlikely event that the display version can not be retrieved, it is set to \"inaccessible\".",
+      "type": "string"
+    },
+    "architecture": {
+      "description": "The architecture of the device, (e.g. \"arm\", \"x86\").",
+      "type": "string"
+    },
+    "client_id": {
+      "description": "A UUID uniquely identifying the client.",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "type": "string"
+    },
+    "device_manufacturer": {
+      "description": "The manufacturer of the device the application is running on. Not set if the device manufacturer can't be determined (e.g. on Desktop).",
+      "type": "string"
+    },
+    "device_model": {
+      "description": "The model of the device the application is running on. On Android, this is Build.MODEL, the user-visible marketing name, like \"Pixel 2 XL\". Not set if the device model can't be determined (e.g. on Desktop).",
+      "type": "string"
+    },
+    "first_run_date": {
+      "description": "The date of the first run of the application.",
+      "format": "datetime",
+      "type": "string"
+    },
+    "locale": {
+      "description": "The locale of the application during initialization (e.g. \"es-ES\"). If the locale can't be determined on the system, the value is [\"und\"](https://unicode.org/reports/tr35/#Unknown_or_Invalid_Identifiers), to indicate \"undetermined\".",
+      "type": "string"
+    },
+    "os": {
+      "description": "The name of the operating system. Possible values: Android, iOS, Linux, Darwin, Windows, FreeBSD, NetBSD, OpenBSD, Solaris, unknown",
+      "type": "string"
+    },
+    "os_version": {
+      "description": "The user-visible version of the operating system (e.g. \"1.2.3\"). If the version detection fails, this metric gets set to `Unknown`.",
+      "type": "string"
+    },
+    "telemetry_sdk_build": {
```

Comment:
This appears not to be "documented" in the Glean SDK. @Dexterp37 is this an oversight?

Reply:
You can find the docs for that [here](https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section).

Reply:
Got it. Should this be added here:

https://github.com/mozilla/glean/blob/ccf91a74dc0d95e5b62df2452187157e882f1907/glean-core/metrics.yaml#L32

?

Reply:
Oh right, you're right.